src/Recommendations/Rule/BidirectionalRelationship.php

```diff
@@ -16,6 +16,9 @@
 use Psr\Log\LoggerInterface;
 use Throwable;
 
+/**
+ * TODO: rename into RelatedArticles?
```

Comment:
but when a `$input` comes in, `getRelatedArticles()` which is only allowed for articles. Also `supports()` only lists article content types. I'll try inserting a log an importing api-dummy to see.